### ISSUE TITLE
Adding example API query for the /search endpoint 

### ISSUE THREAD

**Issue by buniello** [opentargets/issues#2673]
Users have been asking for an example GraphQL API query for the /search endpoint.
This example query should be also added to the API page that links out from Platform Home Page.

Example query for insulin:
```
query searchInsulin {
  search(queryString:"insulin"){
    total
    hits {
      id
      entity
      description
    }
  }
}

```

Query to get many entries there are in each entity category for Insulin:
```

query searchInsulinCatCounts {
  search(queryString:"insulin"){
    aggregations {
      total
      entities {
        total
        name
      }
    }
  }
}

```

**Comment by LucaFumis** [comment]
Preview available here: https://deploy-preview-39--ot-platform.netlify.app/api

Might want to adjust description text and where in the list of examples we want it.

**Comment by buniello** [comment]
Thank you @LucaFumis, looks good - and I believe the description text and list of examples are also good to go as they are.

**Comment by LucaFumis** [comment]
This is done, so will be included with release 22.09
